libs/stats/odc/stats/_gm_ls_bitmask.py

```diff
@@ -0,0 +1,175 @@
+"""
+Landsat QA Pixel Geomedian
+"""
+from functools import partial
+from typing import Optional, Tuple
+
+import dask.array as da
+import xarray as xr
+import numpy as np
+from odc.algo import geomedian_with_mads, keep_good_only, erase_bad, to_rgba
+from odc.algo._masking import _xr_fuse, _first_valid_np, mask_cleanup, _fuse_or_np
+from odc.algo.io import load_with_native_transform
+from odc.stats.model import Task
+
+from . import _plugins
+from .model import StatsPluginInterface
+
+
+class StatsGMLSBitmask(StatsPluginInterface):
+    NAME = "gm_ls_bitmask"
+    SHORT_NAME = NAME
+    VERSION = "0.0.1"
+
+    def __init__(
+            self,
+            bands: Optional[Tuple[str, ...]] = None,
+            mask_band: str = "QA_PIXEL",
+            filter: Optional[Tuple[int, int]] = None,
+            aux_names=dict(smad="sdev", emad="edev", bcmad="bcdev", count="count"),
+            resampling: str = "nearest",
+            work_chunks: Tuple[int, int] = (400, 400),
+            **other,
+    ):
+        self.mask_band = mask_band
+        self.resampling = resampling
+        self.bands = bands
+        self.filter = filter
+        self.work_chunks = work_chunks
+        self.renames = aux_names
+        self.aux_bands = list(aux_names.values())
+        self.return_SR = False  # Flag to scale USGS Landsat bands into surface reflectance
```

Comment:
I don't think this is the correct terminology.

Can we say "return_scale" maybe? And make it default to `10,000`.